nengo/spa/spa.py

```diff
@@ -88,7 +88,11 @@ def __setattr__(self, key, value):
 
     def get_module(self, name):
         """Return the module for the given name."""
-        return self._modules[name]
+        if name in self._modules:
+            return self._modules[name]
+        elif '_' in name:
+            module, name = name.rsplit('_', 1)
```

Comment:
So this means that we can't have inputs and outputs names with an underscore in them, yes? This should be documented somewhere. (If we use an lsplit here, then we can't use module names with underscores.. haha)